jc/lib.py

```diff
@@ -102,6 +102,8 @@
     'mpstat-s',
     'needrestart',
     'netstat',
+    'net-localgroup',
+    'net-user',
```

Comment:
Is the Windows command `net user` or `net users`? If `net users`, then the parser name should be `net-users`.

Reply:
Leaving as is. see [https://github.com/kellyjonbrazil/jc/pull/602#discussion_r1848938465](reference)

Reply:
Sounds good, thanks!